irohad/main/CMakeLists.txt

```
@@ -62,6 +62,13 @@ target_link_libraries(irohad
     rapidjson
     keys_manager
     common
+    iroha_conf_loader
     )
 
+add_library(iroha_conf_loader iroha_conf_loader.cpp)
```

```suggestion
add_library(iroha_conf_loader 
  iroha_conf_loader.cpp)
```
This formatting should be in other changes as well

---

I see that throughout the project the 1-line syntax is used in such cases.